### DISCUSSION TITLE
Add ability for wallet-only mode GUI users to add peers like full node mode users can in GUI (side quest: a checkbox to make that peer trusted)

### DISCUSSION THREAD

**Discussion by OverActiveBladderSystem** [Chia-Network/chia-blockchain-gui#2369] [Ideas]
When helping a wallet-only GUI user resolve their issue of no peers, the only way to add peers manually was through the CLI which is not easy for the average GUI user.

Running the full node mode allows manual addition of peers in the GUI, but there is no way to manually add peers in the GUI for wallet-only mode.

Side Quest, allow users the ability to specify (checkbox) the manually entered peer is trusted and only use that single peer for wallet connections. (So pools and/or other providers can offer a better wallet-only service, with less resources needed, vs 3 random peers?)

Bonus Points, make the sync icons move to visually indicate they are actively working and/or show block height its syncd to on hover (and what height is peak) to give the user some feedback that "something" is actually happening or how far they have to go.

**Comment by AbandonedLand** [comment]
I run a lot of different VMs that are both full nodes and wallet only.  It would be great to point all my wallets at my full node as it's soul source of truth. So I didn't have to have full nodes on multiple machines. (These are non-farming full nodes)

**Reply by BrandtH22** [reply]
@AbandonedLand , you can absolutely point multiple wallets to a single node using the instructions here:
https://docs.chia.net/faq/#what-are-trusted-peers-and-how-do-i-add-them

Note you will likely want to do the optional step that directly assigns the full node IP as the wallets full node peer. 
Let us know if any questions arise :+1:

**Comment by BrandtH22** [comment]
@OverActiveBladderSystem , note I transferred this enhancement request to the GUI repo for better visibility 👍
